handler.js

```diff
@@ -340,3 +355,10 @@ function getObjectDefault(obj, key, defaultValue) {
     var value = obj[key];
     return value != undefined ? value : defaultValue;
 }
+
+function getKeyValuePairs(str, del = ';'){
+    if(str){
+      const querystring = require('querystring');
```

Comment:
I think it would be better to include this globally, at the top of the file, with the other `require` lines...

Reply:
ok. I moved it to the top.